pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -54,7 +54,7 @@ stdenv.mkDerivation rec {
   buildInputs =
     [ zlib glib perl pixman
       vde2 texinfo makeWrapper lzo snappy
-      gnutls nettle curl
+      gnutls nettle curl libslirp
```

Comment:
Is this a strictly required dependency of some sorts? If not please give it a feature flag that defaults to `!nixosTestRunner` like with many of the features above.

Reply:
Oh, I see, we've been using the vendored version previously. So this is likely okay to stay as is.

Reply:
It does seem to be possible to completly disable libslirp but as we've already been using the vendored copy I'd rather just swap them out.

Reply:
Yeah that is reasonable. This actually also shrinks the closure by 0.1MB ;)